src/server/server.js

```diff
@@ -39,6 +39,12 @@ app.get('/api/metas', async (req, res) => {
   res.send(metasCache.data);
 });
 
+app.get('/refresh_metas', (req, res) => {
+  metasCache.data = null;
+  metasCache.lastUpdated = null;
+  res.redirect('/');
```

Comment:
this is OK but results in you navigating away from your current page, how about just `res.end()` (which should send a `200` response) and then you can `fetch`/force refresh the current page?